PORTAL/jobs/_job.py

```diff
@@ -233,8 +233,8 @@ def _build_send_script(
 
         queue_log = _utils.quote_shell_str(queue_log)
 
-        reqdir = _utils.quote_shell_str(pfiles.request.root)
-        results_meta = _utils.quote_shell_str(pfiles.result.metadata)
+        _utils.check_shell_str(pfiles.request.root)
+        _utils.check_shell_str(pfiles.result.metadata)
```

Comment:
I'm pretty sure we don't need these any more; their use was replaced by `pushfiles` and `pullfiles`.

```suggestion
```